stories/Filter.stories.js

```diff
@@ -0,0 +1,177 @@
+import VsCheckbox from '@/components/checkbox/Checkbox.vue';
+import VsFilter from '@/components/filter/VsFilter.vue';
+import VsFilterPanel from '@/components/filter/components/VsFilterPanel.vue';
+import VsFilterSection from '@/components/filter/components/VsFilterSection.vue';
+import {
+    VsCol,
+    VsContainer,
+    VsRow,
+} from '@/components/grid';
+
+export default {
+    component: VsFilter,
+    title: 'Filter',
+};
+
+const Template = (args) => ({
+    components: {
+        VsCheckbox,
+        VsCol,
+        VsContainer,
+        VsFilter,
+        VsFilterSection,
+        VsFilterPanel,
+        VsRow,
+    },
+    setup() {
+        return {
+            args,
+        };
+    },
+    template: `
+        <VsContainer>
+            <VsRow>
```

Comment:
Could this maybe be removed and just replaced with a set width for storybook? The view is cut off